config/scorecard/patches/basic.config.yaml

```diff
@@ -4,7 +4,7 @@
     entrypoint:
     - scorecard-test
     - basic-check-spec
-    image: quay.io/operator-framework/scorecard-test:v1.5.0
+    image: quay.io/operator-framework/scorecard-test:v1.22.0
```

Comment:
Should we use tag `v1.22.2` to match our current `operator-sdk` version? 

https://quay.io/repository/operator-framework/scorecard-test?tab=tags&tag=v1.22.2

Reply:
Good catch.